README.md

```diff
@@ -123,6 +127,24 @@ Use this header in an Objective-C project by adding
 
 to your implementation or header files. Note that spaces in your project name result in underscores. So `Your Project` becomes `Your_Project-Swift.h`.
 
+## Branches
+
+### Master
+
+Contains the latest stable release of the vision library
+
+### Develop
+
+The bleeding edge branch. Contains features actively in development and might be unstable
+
+### Swift-2.3
+
+Swift 2.3 support.
+
+### Swift 3
```

Comment:
This is obsolete, as we already communicated according links to clients. We'll tackle this again, when moving Swift 3 to a more prominent place in the repository.